discussion-solutions.md

```diff
@@ -0,0 +1,89 @@
+
+Given a specific time, t, half-life can make a very strong difference on the
+final activity of two radionuclides with the same initial concentration. 
+If you have a very short half-life and the time is much longer than several
+half-lives, the final activity may be very small. 
+
+**What happens if the length of time you wait is very different from the time of your half life?**
+
+If the length of time elapsed to calculate activity differs strongly from your
+half-life (such as by 10x or more), then you will either see a final activity
+that is either very very small (less than 1%) compared to your initial activity (if t >>
+t_(1/2)). Or if (t<<t_(1/2)) then the final activity will likely be very close
+to your initial activity. 
+
+**What other functions could you write that would make this function easier to use?**
+
+## Objective 2:
+
+**How does changing the distance away from the source affect the total particle rate hitting the body?**
+
+Moving away from the source will decrease the particle rate hitting the body by
+a factor of r^2. 
+
+**Imagine if Michael Jordan and Simone Biles were both in the room with this source. Would the particle rate that each of them sees be different? How would it differ?**
+
+We assumed in our exercise that the cross-sectional area of the human body
+being exposed is the person's `height` times the person's `width` across their
+shoulders. Michael Jordan is significantly larger in both directions (210 cm by
+58cm),
+whereas Simone Biles is substantively smaller (142 cm by 38cm). As a result, Simone Biles'
+body will see a lower particle rate incident on her body than Michael Jordan if
+they are the same distance from the source. 
+
+**What would happen if instead of facing the source, you turned sideways so your shoulder was pointing to it?**
+
+If you turn 90 degrees from your intial position facing the source, your
+exposure area would change from `height` times `shoulder width` to `height`
+times `body depth`, which is probably the distance from your chest to your
+spine (through your body). This dimension will be much smaller than the
+shoulder width, so you would be lowering the number of particles incident on
+your body. 
+
+**Do you think there are assumptions to this equation? What are they? Hint: what happens if you make the distance from the source very small?**
+
+There are many good answers to this question, but we will provide a few:
+* The equation is assuming the source can be approximated as a point. If not,
+  the exposure area at a distance r may not be adequately represented as
+  `4*pi*r^2`. 
+* If the distance is very small, the area of the person may become larger than
+  the surface area of the sphere surrounding the source. This is because we
+  assume the source is at a point. If a human is too close, then their area
+  cannot be projected onto a sphere with the assumptions of this equation.
```

Comment:
We should probably add another common assumption here that we are ignoring the attenuation by the background material.